Tests/ApolloCodegenTests/ApolloCodegenTests.swift

```diff
@@ -2,8 +2,7 @@
 //  ApolloCodegenTests.swift
 //  ApolloCodegenTests
 //
-//  Created by Ellen Shapiro on 10/7/19.
-//  Copyright © 2019 Apollo GraphQL. All rights reserved.
+//  Copyright © 2021 Apollo GraphQL. All rights reserved.
```

Comment:
If you'd prefer you can just nuke the headers on the tests like we have for everything in the main lib

Reply:
Yeah, I definitely prefer not having headers on every file.